From d5c4b8347983023c3f59c062de60a203e4144d6f Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Mon, 22 Jul 2019 18:16:39 +0800
Subject: [PATCH] Makefile.am: Update CODE_COVERAGE_RULES for new
 autoconf-archive

According to:
http://git.savannah.gnu.org/gitweb/?p=autoconf-archive.git;a=commitdiff;h=b5a2d69f7a7e0133733673586231b88464a98d58

The usage has been changed, so update it to fix compile error with
autoconf-archive 2019.01.06:

| DEBUG: Executing shell function do_compile
| NOTE: make -j 128
| Makefile:14636: *** missing separator.  Stop.
| ERROR: oe_runmake failed

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index d78d23f..7ebfad2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -42,7 +42,7 @@ noinst_PROGRAMS =
 
 ### Add ax_* rules ###
 # ax_code_coverage
-@CODE_COVERAGE_RULES@
+include $(top_srcdir)/aminclude_static.am
 
 # ax_doxygen
 @DX_RULES@
-- 
2.7.4

